dipy/sims/voxel.py

```diff
@@ -288,7 +299,265 @@ def multi_tensor(gtab, mevals, S0=100, angles=[(0, 0), (90, 0)],
                                                  snr=None)
 
     return add_noise(S, snr, S0), sticks
+    
+def multi_tensor_dki(gtab, mevals, S0=100, angles=[(0., 0.), (90., 0.)],
+                         fractions=[50, 50], snr=20):
+    
+    r"""Simulate the elements of the diffusion and diffusion kurtosis tensor as
+    well as the diffusion signal S based on the DKI model (i.e. taylor 
+    accumulante larger than the fourth order are ignored). Simulations are 
+    based on multicompartmental models which assumes that tissue are described
+    by impermeable diffusion compartments characterized by their only diffusion
+    tensor.
+
+    Parameters
+    -----------
+    gtab : GradientTable
+    mevals : array (K, 3)
+        eigenvalues of the diffusion tensor for each individual compartment
+    S0 : float
+        Unweighted signal value (b0 signal).
+    angles : array (K,2) or (K,3)
+        List of K tensor directions of the diffusion tensor of each compartment
+        in polar angles (in degrees) or unit vectors
+    fractions : float
+        Percentage of the contribution of each tensor. The sum of fractions
+        should be equal to 100%.
+    snr : float
+        Signal to noise ratio, assuming Rician noise.  If set to None, no
+        noise is added.
+
+    Returns
+    --------
+    dt : (6,)
+        elements of the diffusion tensor.
+    kt : (15,)
+        elements of the kurtosis tensor
+    S : (N,) ndarray
+        Simulated signal based on the dki model.
+            Note: This simulated signal ignores the DW taylor accumulantes 
+                  higher than the 4th order. This simulations are useful to 
+                  test dki fit procedures, since tensors fitted on S have to 
+                  match the diffusion and kurtosis tensor produced here. For 
+                  simulations of S with the higher accumulates please refer to 
+                  the multi_tensor function.
 
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from dipy.sims.voxel import multi_tensor_dki
+    >>> from dipy.data import get_data
+    >>> from dipy.core.gradients import gradient_table
+    >>> from dipy.io.gradients import read_bvals_bvecs
+    >>> fimg, fbvals, fbvecs = get_data('small_101D')
+    >>> bvals, bvecs = read_bvals_bvecs(fbvals, fbvecs)
+    >>> gtab = gradient_table(bvals, bvecs)
+    >>> mevals=np.array(([0.0015, 0.0003, 0.0003],[0.0015, 0.0003, 0.0003]))
+    >>> e0 = np.array([1, 0, 0.])
+    >>> e1 = np.array([0., 1, 0])
+    >>> dt, kt, S =  multi_tensor_dki(gtab, mevals)
+    
+    References
+    ----------
+    .. [1] R. Neto Henriques et al., "Exploring the 3D geometriy of the 
```

Comment:
same typo I reported down below, I'm reading backward/going up cause I'm like that.